internal/compiler/generator/rust.rs

```diff
@@ -393,15 +401,15 @@ fn generate_public_component(llr: &llr::PublicComponent) -> TokenStream {
             }
 
             fn show(&self) -> core::result::Result<(), slint::PlatformError> {
-                self.window().show()
+                self.0.window_adapter_ref()?.window().show()
             }
 
             fn hide(&self) -> core::result::Result<(), slint::PlatformError> {
-                self.window().hide()
+                self.0.window_adapter_ref()?.window().hide()
             }
 
             fn window(&self) -> &slint::Window {
-                vtable::VRc::as_pin_ref(&self.0).get_ref().window_adapter.get().unwrap().window()
+                self.0.window_adapter_ref().unwrap().window()
```

Comment:
So this panics?

Reply:
Yes. Am unfortunate consequence of the lazy window adapter creation is that it may even happen during a binding evaluation.